src/reducers/index.ts

```diff
@@ -0,0 +1,12 @@
+import { combineReducers } from 'redux';
+import { pokemonList } from './pokemonList';
+import { selectedPokemon } from './selectedPokemon';
+import { favoritesPokemons } from './favoritesPokemons';
+import { isLoading } from './isLoading';
+
+export const Reducer = combineReducers({
+    pokemonList,
+    selectedPokemon,
+    favoritesPokemons,
+    isLoading
```

Comment:
za bardo rozdzieliłes te reducery, wg mnie na tym poziomie za to wszystko powinien odpowiadać pokemonReducer

Reply:
combineReducers oczywiscie może tu zostać, bo zakladamy hipotetycznie, że aplikacja bedzie się rozrastać i bedzie wiecej reducerów